helpers.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const { AUTH_KEYS_KEYS } = require('@mtproto/core');
+
+function extractMessagesFromUpdates({updates, channelId}) {
+  const messages = [];
+  updates
+    // TODO handle update and delete messages
+    .filter(update => update._ === 'updateNewChannelMessage' && update.message.message)
+    .forEach(({message}) => {
+      if (channelId && +message.to_id.channel_id === +channelId) messages.push(message);
```

Comment:
are you using `+` to case these into numbers? If so thats confusing and I'd prefer something more clear. 
```suggestion
      if (channelId && Number(message.to_id.channel_id) === Number(channelId)) messages.push(message);
```

Reply:
sure, but isn't `Number` and a `+` do literally the same thing?